Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/Result.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
+{
+    /// <summary>
+    ///     Returns list of container's blobs.
+    /// </summary>
+    public class Result
+    {
+        /// <summary>
+        ///     Blob's type.
+        /// </summary>
+        public string BlobType { get; set; }
+
+        /// <summary>
+        ///     Blob's URI.
+        /// </summary>
+        public string Uri { get; set; }
+
+        /// <summary>
+        ///     Blob's name.
+        /// </summary>
+        public string Name { get; set; }
```

Comment:
Could we have an example &lt;example&gt; ?
Also, it would be nice to clarify that this potentially can contain the full blob path inside the container (e.g. somefolder/myfile.jpg). At list if I understood correctly.

Reply:
Example added + modified the code so this will be filename only.